File: src/main/java/io/github/Earth1283/aptMc/api/HttpClientProvider.java
package io.github.Earth1283.aptMc.api;

import com.google.gson.Gson;
import java.net.http.HttpClient;

public final class HttpClientProvider {
    public static final HttpClient CLIENT = HttpClient.newHttpClient();
    public static final Gson GSON = new Gson();
    public static final String USER_AGENT = "apt-mc/1.2 (parody-cli)";

    private HttpClientProvider() {}
}

File: src/main/kotlin/io/github/Earth1283/aptMc/api/HttpClientProvider.kt
package io.github.Earth1283.aptMc.api

import com.google.gson.Gson
import java.net.http.HttpClient

object HttpClientProvider {
    @JvmField val CLIENT: HttpClient = HttpClient.newHttpClient()
    @JvmField val GSON: Gson = Gson()
    const val USER_AGENT: String = "apt-mc/1.2 (parody-cli)"
}
